app/templates/test.js

```diff
@@ -1,9 +1,9 @@
 import test from 'ava';
-import fn from '.';
+import m from '.';
```

Comment:
Exactly, thanks for linking it :). If we go for something short (`m` or `fn`), I'm more in favour of `m`. Because most of the time, it's not only a function. For instance, `fn.loadSync` just looks weird while `m.loadSync` is much better.